libexec/dapp/dapp---testnet-launch

```diff
@@ -81,17 +88,17 @@ echo >&2 "dapp-testnet:  Chain ID: $CHAINID"
 echo >&2 "dapp-testnet:  Database: $chaindir"
 echo >&2 "dapp-testnet:  Geth log: $chaindir/geth.log"
 
-echo "$address"     > "$chaindir/config/account"
-echo "$ETH_RPC_URL" > "$chaindir/config/rpc-url"
-echo "$port"        > "$chaindir/config/node-port"
+echo "${address[@]}" > "$chaindir/config/account"
```

Comment:
I think a line per account would be nice, like with `printf "%s\n" "${address[@]}"`.

Reply:
on it!